Corona-Warn-App/src/main/res/values-tr/legal_strings.xml

```diff
@@ -220,4 +220,13 @@
     <!-- XTXT: DCC Ticketing: Consent Two: Legal: Point 2-4 -->
     <string name="dcc_ticketing_consent_two_legal_point_2_4" translatable="false">"Sağlayıcının sertifikaya ilişkin talepleri (örn. hangi aşılar veya test yöntemleri kabul edilir)"</string>
 
+    <!-- DCC Reissuance Consent Screen -->
+    <!-- XTXT: DCC Reissuance legal header-->
+    <string name="dcc_reissuance_legal_header" translatable="false">„Onay“ üzerine tıklayarak aşağıdaki adımları kabul edersiniz:</string>
+    <!-- XTXT: DCC Reissuance legal first bullet point-->
+    <string name="dcc_reissuance_legal_one" translatable="false">Uygulama, kontrol amaçlı olarak aşı sertifikasını RKI’ya iletir.</string>
+    <!-- XTXT: DCC Reissuance legal second bullet point-->
+    <string name="dcc_reissuance_legal_two" translatable="false">RKI aşı sertifikasını kontrol eder ve güncel bir sertifika oluşturur ve bunu uygulamanıza aktarır.</string>
+    !-- XTXT: DCC Reissuance button text-->
+    <string name="dcc_reissuance_legal_button" translatable="false">Onay</string>
```

Comment:
To be consistent with other consent screens, we should leave the button as it is , but we should change the legal text to be` „Kabul et“ üzerine tıklayarak aş ...`

Reply:
I do not think we can change legal text as it is coming from the lawyers

Reply:
You can align with Martin ofc, but last time he said for this case it is fine , see https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11109?focusedCommentId=1254855&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1254855

Reply:
will do it in another PR